pkg/k8shandler/clusterloggingrequest.go

```diff
@@ -35,6 +35,12 @@ func (clusterRequest *ClusterLoggingRequest) Get(objectName string, object runti
 	return clusterRequest.client.Get(context.TODO(), namespacedName, object)
 }
 
+func (clusterRequest *ClusterLoggingRequest) GetClusterResource(objectName string, object runtime.Object) error {
+	namespacedName := types.NamespacedName{Name: objectName}
+
+	return clusterRequest.client.Get(context.TODO(), namespacedName, object)
```

Comment:
Are there docs that say that you omit the namespace name from the object to get a cluster scoped object?

Reply:
I didnt find any.  i simply guessed

Reply:
ok - we'll know soon enough if it works - but something to keep in mind that we are using an unknown and/or undocumented api feature that may break unexpectedly in the future

Reply:
@jcantrill did you see it came back with the correct object during your testing with `debug` statements?